### PR TITLE
Woptim/update uberenv

### PR DESCRIPTION
**Pull request by adrienbernede**
This PR is built on top of #55 to complete the porting of radiuss-spcak-configs to spack environments for the new Uberenv.

This PR should be updated with the latest changes we are currently making to radiuss-spack-configs before it is merged into main.

### CODE REVIEW

File: blueos_3_ppc64le_ib/spack.yaml [hunk 1]
```diff
@@ -0,0 +1,482 @@
...
+      environment: {}
+      extra_rpaths: []
+  - compiler:
+      spec: clang@10.0.1
+      paths:
+        cc: /usr/tce/packages/clang/clang-10.0.1/bin/clang
+        cxx: /usr/tce/packages/clang/clang-10.0.1/bin/clang++
+        fc: /usr/tce/packages/xl/xl-2020.09.17/bin/xlf2003_r
+        f77: /usr/tce/packages/xl/xl-2020.09.17/bin/xlf_r
+      flags: {}
```
Comment (white238) [inline blueos_3_ppc64le_ib/spack.yaml:126]: Is there a reason none of these clang specs point at the non-default gnu standard library via `--gcc-toolchain`? The system default on blueosand toss3 is 4.9.3

Comment (adrienbernede) [reply blueos_3_ppc64le_ib/spack.yaml:126]: The present PR is based upon the current state of radiuss-spack-configs: we used to define the gcc-toolchain in the spec.

I am changing that in https://github.com/LLNL/radiuss-spack-configs/pull/49, which is a major update for us. We still have some issues, and the current maintenance shower is not helping with the testing.

File: blueos_3_ppc64le_ib/spack.yaml [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+      - spec: spectrum-mpi@10.3.1.03rtm0%xl@beta2019.06.20
+        prefix: /usr/tce/packages/spectrum-mpi/spectrum-mpi-rolling-release-xl-beta-2019.06.20
+      buildable: false
+    python:
+      buildable: false
+      version: [3.8.2]
+      externals:
+      - spec: python@3.8.2
+        prefix: /usr/tce/packages/python/python-3.8.2
+  compilers:
```
[the remark this answers is not in the document]
Comment (adrienbernede) [reply blueos_3_ppc64le_ib/spack.yaml:79]: I cannot hurt indeed.

Comment (adrienbernede) [inline blueos_3_ppc64le_ib/spack.yaml:79]:
```suggestion
  compilers::
```

File: blueos_3_ppc64le_ib/spack.yaml (hunk 1 above)
Comment (adrienbernede) [reply blueos_3_ppc64le_ib/spack.yaml:126]: Note that I created this to test that the new env would be working for us.
In practice, I will likely merge #49 first, and then redo the above.

**Pull request closed without merging**